Sources/ProjectSpec/SpecValidation.swift

```diff
@@ -159,7 +159,7 @@ extension Project {
                         if getProjectTarget(dependency.reference) == nil {
                             errors.append(.invalidTargetDependency(target: target.name, dependency: dependency.reference))
                         }
-                    case .project(let dependencyProjectName):
+                    case .project(let dependencyProjectName), .package(let dependencyProjectName):
```

Comment:
It is better to not check package by `getProjectReference`, it is out of range for `getProjectReference` and not preferred usage even it can check.  Please check by right way to check package or create  `getSwiftPackageReference`.

Reply:
I updated this in the latest version, I hope this resolves this issue.